samples/cxx/jacobian/derivative_speed.cpp

```diff
@@ -17,9 +17,7 @@
 #include <iostream>
 #include <iomanip>
 #include <numeric>
-#include "cantera/base/Solution.h"
-#include "cantera/thermo/IdealGasPhase.h"
-#include "cantera/kinetics.h"
+#include "cantera/core.h"
```

Comment:
I think the include graph for `core.h` already gets `cantera/numerics/eigen_sparse.h` (included below).

Reply:
indeed!